### PR TITLE
Execute overhaul

### LINKED ISSUE #56: create/rename should do argument checking
```
create_class Class with Spaces
```

adds "Class" but does not report an error diagnostic to the user

```
rename_class Class Class2 with spaces
```

renames "Class" to "Class2" but does not report an error diagnostic to the user

```
create_attribute Class attribute with spaces
```

creates an attribute called "attribute" but does not report an error diagnostic to the user

and so on...

### LINKED ISSUE #99: Execute Function Overhaul
- Change execute function to be generic so that there is less redundant code
- And so that we can easily evaluate the number of arguments. 

Size-estimate: 1 
Primary: Amy 
Secondary:

### PR DESCRIPTION
**Pull request by 16aburnett**
Changed the execute function to generically call functions to remove redundancy. 

Also added argument handling 

Included better error messages and added red color to error messages. 

Updated CommandData with functions and argument counts. 

Closes #99 Closes #56

### CODE REVIEW

File: code/UMLEditor.py [hunk 1]
```diff
@@ -13,14 +13,16 @@
 from UMLModel import UMLModel
 import UMLClass
 import UMLRelationship
-from CommandData import COMMANDS
+import CommandData
```
Comment (16aburnett) [inline code/UMLEditor.py:16]: I changed this to get around a circular import

File: code/UMLEditor.py [hunk 2]
```diff
@@ -13,14 +13,16 @@
...
 ##########################################################################
 # Constants 
 
 # Colors for printing variation
-PROMPT_COLOR = "\033[1;36m"
-NORMAL_COLOR = "\033[0;37m"
+PROMPT_COLOR  = "\033[1;36m"
+ERROR_COLOR   = "\033[91m"
+SUCCESS_COLOR = "\033[92m"
+NORMAL_COLOR  = "\033[0;37m"
```
Comment (16aburnett) [inline code/UMLEditor.py:25]: More colors!

File: code/UMLEditor.py [hunk 3]
```diff
@@ -178,20 +156,8 @@ def REPL():
...
-            # Case 1: Command has arguments 
-            else:
-                command, arguments = (words[0],words[1:])
-                # test for create_attribute (does not work at the moment):
-                # UMLModel.create_attribute = (self.attributes, arguments[0], arguments[1])
-                execute(model, command, arguments)
-        else:
-            print("Invalid command, type 'help' for information on commands")
+        # This handles the case where there are no arguments
+        execute(model, words[0], words[1:])
```
Comment (16aburnett) [inline code/UMLEditor.py:160]: I made execute() responsible for checking if a command is valid.

**Review by jdchica2020 — APPROVED** [review]

**Comment by jdchica2020** [comment]
It looks good to me.

**Pull request merged**